### PR TITLE
Update scrive esign 

### PR DESCRIPTION
**Pull request by marcusdelang**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by marcusdelang** [comment]
> @microsoft-github-policy-service agree [company="{your company}"]

@microsoft-github-policy-service agree [company="Scrive AB"]

**Comment by marcusdelang** [comment]
@microsoft-github-policy-service agree company="Scrive AB"

**Comment by marcusdelang** [comment]
```
@microsoft-github-policy-service agree company="Scrive AB"
```

**Comment by marcusdelang** [comment]
Is this validation error really correct?
I am trying to config multi-auth as per your [documentation](https://learn.microsoft.com/en-us/connectors/custom-connectors/multi-auth). Using `connectionParameterSets` should be ok?

File: certified-connectors/Scrive eSign/apiProperties.json [hunk 1]
```diff
@@ -1,64 +1,143 @@
...
-          "x-ms-apimTemplateParameter.name": "customApiUrl",
-          "x-ms-apimTemplateParameter.value": "@connectionParameters('customApiUrl','https://scrive.com')"
+    "connectionParameterSets": {
+      "uiDefinition": {
+        "displayname": "Authentication Type",
+        "description": "Type of authentication to be used."
+      },
+      "values": [
+        {
+          "name": "Email/Password",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Scrive eSign/apiProperties.json:51]: @marcusdelang please change this to "basic-auth"

File: certified-connectors/Scrive eSign/apiProperties.json [hunk 2]
```diff
@@ -1,64 +1,143 @@
...
-    "publisher": "Scrive AB",
-    "stackOwner": "Scrive AB",
-    "iconBrandColor": "#27282D",
-    "capabilities": [
-      "triggers",
-      "actions"
-    ],
     "connectionParameters": {
       "username": {
         "type": "securestring",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Scrive eSign/apiProperties.json:5]: @marcusdelang please change to "string"

File: certified-connectors/Scrive eSign/apiProperties.json [hunk 3]
```diff
@@ -1,64 +1,143 @@
...
+        {
+          "name": "OAuth",
+          "uiDefinition": {
+            "displayname": "OAuth",
+            "description": "OAuth authentication."
+          },
+          "parameters": {
+            "token": {
+              "oAuthSettings": {
+                "clientId": "b656c82f946739f0_36551",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Scrive eSign/apiProperties.json:107]: @marcusdelang secret look alike values are not allowed. Please replace with dummy value.

File: certified-connectors/Scrive eSign/apiProperties.json [hunk 4]
```diff
@@ -1,64 +1,143 @@
...
+                }
+              },
+              "type": "oauthSetting"
+            }
+          }
         }
-      }
+      ]
+    },
+    "publisher": "Scrive AB",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Scrive eSign/apiProperties.json:136]: @marcusdelang please add the stackOwner property 

    "stackOwner": "Name of the owner of the service or product."

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello @marcusdelang thank you so much for the fix. Please allow us some time to test this.

**Review by Amjed-Ayoub — APPROVED** [review]

**Pull request merged**